weldx/asdf/tags/weldx/core/netcdf.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+from pandas.api.types import is_datetime64_any_dtype as is_datetime
+
+from weldx.asdf.types import WeldxType
+
+
+class NetCDFAttribute:
+    def __init__(self, name, data):
+        self.name = name
+        self.data = data
```

Comment:
This looks like a perfect `@dataclass` application 😉 
(same for other NetCDF classes)